### PR TITLE
Allow more fine grained selection of projects and sorting

### PR DESCRIPTION
**Pull request by txels**
Work on #75 

Done:
- Added possibility to filter subsets of projects in sprintly and github, and choose display order in feature list

Pending:
- Tests
- Maybe rethink the sorting criteria in feature list: currently they are sorted by provider, and within provider there is some implicit sorting. It may be that I want to list first PRs and stories from given repos (intertwined providers) or by priority within repos... I may create a proper story for this

### CODE REVIEW

File: deploystream/providers/github/__init__.py [hunk 1]
```diff
@@ -33,13 +33,29 @@ def __init__(self, token, organization=None, **kwargs):
...
+        }
+
+        if repositories:
+            filtered = []
+            for entry in repositories:
+                if entry in all_repos:
+                    filtered.append(all_repos[entry])
+                else:
+                    print('repo {0}/{1} not found in github'.format(*entry))
+            self.repositories = filtered
```
Comment (txels) [inline deploystream/providers/github/__init__.py]: This is a preliminary implementation.
I pushed for you to see what I was aiming at (choose and sort repos).

Comment (txels) [inline deploystream/providers/github/__init__.py]: discarded in favour of the one in story 70

File: deploystream/providers/sprintly/__init__.py [hunk 2]
```diff
@@ -54,8 +57,8 @@ def get_features(self, **filters):
         """
         features = []
 
-        for project in self.projects:
-            feature_endpoint = self.api.products[project.id].items
+        for project_id in self.projects:
+            feature_endpoint = self.api.products[project_id].items
             for criterion in self.current:
                 features += feature_endpoint(**criterion)
```
Comment (alexcouper) [inline deploystream/providers/sprintly/__init__.py:64]: Distinct lack of tests for this...but merging anyhow.

Comment (txels) [inline deploystream/providers/sprintly/__init__.py:64]: Ah yes there was a comment in the PR header. I forgot I probably wanted to do that before merge :-O

Comment (txels) [inline deploystream/providers/sprintly/__init__.py:64]: Well, this code in particular is actually covered by tests.
But will write some more to check sorting.

**Pull request merged**